.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.3
```

Comment:
This is for rbenv, right? Do we even have anyone using that? @vkrizan perhaps?

Reply:
I do